pkg/metrics/exporters/stackdriver/stackdriver.go

```diff
@@ -31,26 +36,22 @@ func Start(ctx context.Context) error {
 		return err
 	}
 
-	exporter, err := stackdriver.NewExporter(stackdriver.Options{
-		MetricPrefix:      metricPrefix,
-		MonitoredResource: monitoredresource.Autodetect(),
-	})
```

Comment:
the updated code is not quite right given https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/main/exporter/metric/option.go#L77-L86

I don't think we need to set a project ID as it will get auto detected per the docs. 

What we do want to set is the metrics prefix and that API seems to be a little different from the options:

https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/main/exporter/metric/option.go#L104-L112

Here's a quick example I found on [sourcegraph](https://sourcegraph.com/github.com/GoogleContainerTools/skaffold@aec8e3e78beaeb874b7c08b06ff4514b68c225a8/-/blob/pkg/skaffold/instrumentation/export.go?L146-155)